compiler/ballerina-parser/src/test/java/io/ballerinalang/compiler/parser/test/syntax/expressions/AccessExpressionsTest.java

```diff
@@ -167,4 +167,9 @@ public void testOptionalFieldAccessWithMissingExpression() {
     public void testMemberAccessRecovery() {
         testFile("access-expr/member_access_expr_source_11.bal", "access-expr/member_access_expr_assert_11.json");
     }
+
+    @Test
+    public void testModuleCallSyntaxErrors() {
+        testFile("access-expr/module_call_source.bal", "access-expr/module_call_assert.json");
+    }
```

Comment:
Let's move this test to array type desc tests and change the names accordingly

Reply:
bal file name -> `array_type_source_21.bal`
source file name -> `array_type_assert_21.json`
testName -> `testInvalidArrayBaseExpression`

Reply:
+1

Reply:
Okay. I'll move.